### ISSUE TITLE
Tick error (on zone?)

### ISSUE THREAD

**Issue by FinalDoom** [FinalDoom/FFXI-FishingForm#32]
**[Original report](https://bitbucket.org/FinalDoom/ffxi-fishingform/issue/32) by FinalDoom (Bitbucket: [FinalDoom](https://bitbucket.org/{d7bd602f-3443-4a67-a621-ea4c07d584c2}), GitHub: [FinalDoom](https://github.com/FinalDoom)).**

----------------------------------------

```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at Fishing.FishingForm.UpdateInfo() in FishingFormsource\FishingForm.cs:line 2240
   at Fishing.FishingForm.timer_Tick(Object sender, EventArgs e) in FishingFormsource\FishingForm.cs:line 3401
   at System.Windows.Forms.Timer.OnTick(EventArgs e)
   at System.Windows.Forms.Timer.TimerNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**Comment by FinalDoom** [comment]
**Original comment by FinalDoom (Bitbucket: [FinalDoom](https://bitbucket.org/{d7bd602f-3443-4a67-a621-ea4c07d584c2}), GitHub: [FinalDoom](https://github.com/FinalDoom)).**

----------------------------------------

Until this happens again for new source line numbers, can't fix. It shouldn't do this as is.

**Comment by FinalDoom** [comment]
**Original comment by FinalDoom (Bitbucket: [FinalDoom](https://bitbucket.org/{d7bd602f-3443-4a67-a621-ea4c07d584c2}), GitHub: [FinalDoom](https://github.com/FinalDoom)).**

----------------------------------------

Updated with current lines

**Comment by FinalDoom** [comment]
**Original comment by FinalDoom (Bitbucket: [FinalDoom](https://bitbucket.org/{d7bd602f-3443-4a67-a621-ea4c07d584c2}), GitHub: [FinalDoom](https://github.com/FinalDoom)).**

----------------------------------------

[fix] should fix #32, also deduplicate chat code
